src/services/requestService.js

```diff
@@ -28,4 +30,47 @@ export default class RequestService {
       where: { requesterId: id }
     });
   }
+
+  /**
+    * Get Request by id and status
+    * @param {object} id - the id of assigned manager || requester.
+    * @param {object} statusId - the status of the request.
+    * @return {Promise<object>} A promise object with role detail.
+    * @memberof RequestService
+    */
+  static async getRequest(id, statusId) {
+    const requests = await Request.findOne({
+      include: [
+        {
+          model: User,
+          as: 'requester',
+          attributes: ['firstName', 'lastName']
+        }],
+      where: { statusId }
+    });
+    if (requests.length === 0) throw new ApiError(404, 'No such request');
+    return requests.dataValues;
+  }
+
+  /**
+   * Function for update query
+   *
+   * @param {Object} updateValues - Object of fields to be updated
+   * @param {string} obj - An object of the keys to be searched e.g {id}, {email}
+   * @memberof RequestService
+   * @returns {Promise<object>} A promise object with user detail.
+   */
+  static async updateAnyRequest(updateValues, obj) {
+    try {
+      const result = await Request.update(
+        updateValues,
+        { where: obj, returning: true }
+      );
+      const [bool, [user]] = result;
```

Comment:
Please change the variable name `bool` to make it more descriptive, `updated` can be considered